### DISCUSSION TITLE
What Zig version do you use to compile this project?

### DISCUSSION THREAD

**Discussion by sweetbbak** [malcolmstill/foxwhale#71] [Q&A]
pretty much the title. Im having some issues with zig 0.14.0-dev and zig 0.13.0 regarding an error with union types.

**Comment by malcolmstill** [comment]
@sweetbbak the intention was to keep up to date with zig `master` (so zig-0.14.0-dev at the moment) and I made some changes to that effect last week https://github.com/malcolmstill/foxwhale/commit/e70857fdc44f6d345771ba9ee2dca7acdd1f80d6 ...maybe there have been some breaking changes in zig `master` in the past week that need to be fixed?

**Reply by malcolmstill** [reply]
I'll see if I can take a look today or tomorrow..

**Reply by sweetbbak** [reply]
ah I see, I was not on the latest 0.14.0... so thats my bad. It built perfectly with the latest dev version. Really cool project btw, I'm learning a lot reading through the code... so thank you!
